OnlineLibrary.Common/Pagination/PageOfEntities.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OnlineLibrary.Common.Pagination
+{
+    public class PageOfEntities<T>
```

Comment:
Rename it to PaginatedList with 2 props

int TotalCount
List<T> Results

add constructor that accepts such parameters